atomic_reactor/tasks/common.py

```diff
@@ -40,6 +40,7 @@ class TaskParams:
     pipeline_run_name: str
     user_params: Dict[str, Any]
     task_result: Optional[str]
+    platforms_result: Optional[str]
```

Comment:
The param can't be in the common `TaskParams`, only the prebuild task has it

Reply:
it is optional here, so it doesn't matter, and it we don't have to create new subclass PreBuildTaskParams ......

Reply:
I'm pretty sure you'll get `TypeError: ... missing 1 required positional argument: 'platforms_result'` in every task except prebuild, but I'll check

Reply:
Indeed
```bash
$ atomic-reactor task --build-dir build/ --context-dir context/ --user-params-file user_params.json --namespace idk --pipeline-run-name idk clone
TypeError: BinaryBuildTaskParams.__init__() missing 1 required positional argument: 'platforms_result'
```

Reply:
fixed